starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/GroceryItem.java

```diff
@@ -0,0 +1,48 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by jonlieblich on 10/25/16.
+ */
+
+public class GroceryItem {
+    private String mName, mDescription, mType, mPrice;
+
+    public GroceryItem(String name, String description, String type, String price) {
+        mName = name;
+        mDescription = description;
+        mType = type;
+        mPrice = price;
+    }
+
+    public String getName() {
+        return mName;
+    }
+
+    public void setName(String name) {
+        mName = name;
```

Comment:
Totally fine to have setters here, but I just want to make sure you know you don't always need to have setters. Sometimes you don't want an object's values to change after it's created, so you might prefer not to provide any setters.